src/TrueLayer/Payouts/IPayoutsApi.cs

```diff
@@ -10,6 +10,7 @@ namespace TrueLayer.Payouts
         Pending,
         Authorized,
         Successful,
```

Comment:
There is no successful status for payouts (replaced with executed), can we remove this? Do we have any clients using this atm?

Reply:
> Do we have any clients using this atm?

Not really sure. We need to investigate.

> can we remove this?

I've created an [issue](https://github.com/TrueLayer/truelayer-dotnet/issues/141) so that we can address it in a separate PR.